Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -34,7 +34,7 @@ extension BlockingStrength {
     }
 }
 
-// Additional information shown when the info accessory button is tapped.
+//MARK: Additional information shown when the info accessory button is tapped.
```

Comment:
```suggestion
// MARK: Additional information shown when the info accessory button is tapped.
```
To fix Swiftlint warning